tests/test_suppressions.py

```diff
@@ -55,21 +55,21 @@
     batch_update_findings_full_fixture = json.load(updates_file)
 
 full_findings_fixture = []
-for security_control_id in ['S3.8', 'S3.9', 'S3.14', 'S3.20']:
+for identifier in ['S3.8', 'S3.9', 'S3.14', 'S3.20', 'Inspector']:
     for env in ['dev', 'acc', 'prd']:
```

Comment:
I see this has been added before. @carlovoSBP what is the idea behind these dev/acc/prd envs? Are the current test findings and these new findings anonymized? @fernandogoncalves-me

Reply:
Fixtures for Inspector are anonymized. 

Regarding the different envs, I was also not sure, so just followed the current approach since this is my first contribution to the project.

Reply:
It might be better to not have those environments in there, since this project does not reflect on environments at all. On the other hand the way we work does and I guess this does not hurt.